components/clp-mcp-server/tests/server/test-utils.py

```
@@ -0,0 +1,130 @@
+        {
+            "_id": "test003",
+            "timestamp": 1729267200123,  # Oct 18, 2024 16:00:00.123 (milliseconds)
+            "message": (
+                '{"ts":1729267200123,"pid":1234,"tid":5678,'
+                '"message":"Log with millisecond precision"}\n'
+            ),
+            "orig_file_path": "/var/log/app.log",
+            "archive_id": "abc125",
+            "log_event_ix": 102,
+        },
+    ]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)